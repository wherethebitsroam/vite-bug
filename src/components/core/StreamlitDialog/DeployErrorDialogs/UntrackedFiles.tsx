import React from "react"
import { IDeployErrorDialog } from "./types"

function UntrackedFiles(): IDeployErrorDialog {
  return {
    title: "Unable to deploy app",
    body: (
      <>
        <p>
          This Git repo has untracked files. You may want to commit them before
          continuing.
        </p>
        <p>
          Alternatively, you can either delete the files (if they're not
          needed) or add them to your <strong>.gitignore</strong>.
        </p>
      </>
    ),
  }
}

export default UntrackedFiles
