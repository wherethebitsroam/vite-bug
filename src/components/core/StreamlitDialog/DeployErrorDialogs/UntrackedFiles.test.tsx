import UntrackedFiles from "./UntrackedFiles"

describe("DeployErrorDialog - UntrackedFiles", () => {
  it("should render without crashing", () => {
    const dialog = UntrackedFiles()

    expect(dialog).toMatchSnapshot()
  })
})
