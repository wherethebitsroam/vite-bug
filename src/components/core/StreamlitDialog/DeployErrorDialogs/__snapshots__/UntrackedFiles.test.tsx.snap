// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeployErrorDialog - UntrackedFiles should render without crashing 1`] = `
Object {
  "body": <React.Fragment>
    <p>
      This Git repo has untracked files. You may want to commit them before continuing.
    </p>
    <p>
      Alternatively, you can either delete the files (if they're not needed) or add them to your 
      <strong>
        .gitignore
      </strong>
      .
    </p>
  </React.Fragment>,
  "title": "Unable to deploy app",
}
`;
